src/pybind/mgr/cephadm/services/iscsi.py

```diff
@@ -158,15 +158,15 @@ def post_remove(self, daemon: DaemonDescription, is_failed_deploy: bool) -> None
 
         if 'dashboard' in self.mgr.get('mgr_map')['modules']:
             # remove config for dashboard iscsi gateways
-            ret, out, err = self.mgr.check_mon_command({
+            ret, out, err = self.mgr.mon_command({
                 'prefix': 'dashboard iscsi-gateway-rm',
                 'name': daemon.hostname,
             })
             logger.info(f'{daemon.hostname} removed from iscsi gateways dashboard config')
```

Comment:
```suggestion
        # remove config for dashboard iscsi gateways
        ret, out, err = self.mgr.mon_command({
            'prefix': 'dashboard iscsi-gateway-rm',
            'name': daemon.hostname,
        })
        if not ret:
            logger.info(f'{daemon.hostname} removed from iscsi gateways dashboard config')
```

Would that simplify the code?  I mean we don't need to check that the dashboard module exist, if we don't raise here.